source/non-pattern-templates/for-head.mustache

```diff
@@ -0,0 +1,7 @@
+{{! These are the fonts specified in the the css's @font-face rules. }}
+
+<link rel="preload" href="{{#assetRewrite}}/fonts/249228f0-61ac-40cc-a5a5-5609c9816e3f.woff2{{/assetRewrite}}" as="font">
```

Comment:
Need to include `{{assetsPath}}` at the beginning:

https://github.com/elifesciences/pattern-library/blob/8b52fb3f4df69c2fbb94a5fd6bfe215fca834800/source/_patterns/00-atoms/components/orcid.mustache#L5